modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1438,111 +1440,111 @@ return view.extend({
 					// Probe 802.11r support (and EAP support as a proxy for Openwrt)
 					var has_80211r = L.hasSystemFeature('hostapd', '11r') || L.hasSystemFeature('hostapd', 'eap');
 
-					o = ss.taboption('encryption', form.Flag, 'ieee80211r', _('802.11r Fast Transition'), _('Enables fast roaming among access points that belong to the same Mobility Domain'));
+					o = ss.taboption('encryption', form.Flag, 'ieee80211r', _('Fast transition roaming'), _('Enable 802.11r <abbr title="Fast Transition">FT</abbr> roaming between APs that are part of the same mobility domain.'));
```

Comment:
I dislike moving 802.11r like of explanations to only comment text

Reply:
I humbly disagree, 802.11k, 802.11r, 802.11v and 802.11w are all just  historical amendments to 802.11 to originally defined these that has long since been folded in to later revisions of the standard, 802.11-2016.
In much the same way we do not commonly refer to WMM as "802.11e WMM" these days, we just call it WMM.
Fast Transition (FT) is the name of the feature defined in the 802.11 standard.